rest/model/build.go

```diff
@@ -38,14 +38,36 @@ type APIBuild struct {
 	RevisionOrderNumber int            `json:"order"`
 	TaskCache           []APITaskCache `json:"task_cache,omitempty"`
 	// Tasks is the build's task cache with just the names
-	Tasks             []string             `json:"tasks"`
+	Tasks             []*string            `json:"tasks"`
```

Comment:
I don't have a strong opinion on this, but is this change just to keep it consistent with `Tags`? I feel like returning just `[]string` instead of `[]*string` was nice since the users don't have to account for JSON `null`s within the array.

Reply:
Yeah sort of didn't think about it; i'll change it back